SmartDeviceLink_Example/Classes/ProxyManager.m

```diff
@@ -322,7 +343,7 @@ + (void)sdlex_sendPerformOnlyChoiceInteractionWithManager:(SDLManager *)manager
     }];
 }
 
-+ (SDLSoftButton *)sdlex_softButton1WithManager:(SDLManager *)manager {
+- (void)sdlex_softButton1WithManager:(SDLManager *)manager isImageVisible:(Boolean)imageVisible completionHandler:(void (^)(SDLSoftButton * button))completionHandler {
```

Comment:
This (and others) should be `sdlex_prepareSoftButton1WithManager`

Reply:
Fixed